docs/src/content/integration-testing.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Integration Testing"
+---
+
+# Integration Testing
+
+There are different types of API keys that let you do functional or performance integration testing. You don’t need different Notify accounts or access to other Notify environments.
+
+## Types of API Keys
+
+ |
+--- | --- 
+Live | sends to anyone
+Team and whitelist | limits who you can send to
+Test | pretends to send messages
+
+## Live key
+
+Once your service is live you are able to create Live keys. You can then use these keys to send messages to anyone.
+
+Messages sent with a live key show up on your dashboard and count against your text message and email allowances.
+
+You should revoke and re-create these keys on a regular basis. You can have more than one active key at a time. To revoke a key click the revoke button on the API Key page.
+
+
+You should never send test messages to invalid numbers or addresses using a live key. This will lead to your API key being revoked.
```

Comment:
This should be a quote, no? Compare new and old:

![image](https://user-images.githubusercontent.com/626664/47685765-de889e80-dc2b-11e8-8eab-e9e21c3faaf9.png)

![image](https://user-images.githubusercontent.com/626664/47685773-e9dbca00-dc2b-11e8-8e51-0e5dff79beea.png)